### PR TITLE
populate footer links

### PR DESCRIPTION
**Pull request by qrkourier**

### CODE REVIEW

**Comment by dovholuknf** [comment]
The whole footer needs a do-over. Also it'd be nice to see the icons etc. This PR only serves to make the footer bigger which isn't what we need. Please rework the footer and make it pleasing to look at/use? :)

**Comment by dovholuknf** [comment]
strangely, the preview doesn't show "Blog" anywhere?

**Comment by qrkourier** [comment]
> strangely, the preview doesn't show "Blog" anywhere?

It's added in a different PR.

**Comment by dovholuknf** [comment]
LOL -- i thought I was looking at the blog preview issue. I must have clicked on the wrong one. my bad

**Comment by qrkourier** [comment]
> The whole footer needs a do-over. Also it'd be nice to see the icons etc. This PR only serves to make the footer bigger which isn't what we need. Please rework the footer and make it pleasing to look at/use? :)

I'll take a stab at styling it. Less big sounds doable. 

Re: icons, I considered duplicating the icons from links dropdown but dismissed the idea because it seems to add more undesirable clutter than value, especially considering the monochromatic motif of the footer and the extra work that would be required to create subdued hue icon counterparts for each social link, and the fact that we won't have icons for every item in the footer creates a visual inconsistency between those that have icons and those that don't.

**Comment by qrkourier** [comment]
## Kubernetes

image

## Twilio

image

## Readme

image

## GitHub

image

File: docusaurus/src/css/custom.css [hunk 1]
```diff
@@ -93,11 +98,10 @@ html[data-theme="dark"] {
 }
 
 .button:hover {
-  background: linear-gradient(90deg,#0068f9 0, #fc0147 100%) ;
+  background: linear-gradient(90deg, #0068f9 0, #fc0147 100%);
 }
 
-input[type="radio"]
-{
+input[type="radio"] {
```
Comment (dovholuknf) [inline docusaurus/src/css/custom.css]: please unformat the formatting for this pr. ty

File: docusaurus/docusaurus.config.js [hunk 2]
```diff
@@ -298,23 +298,22 @@ const config = {
...
+            label: 'CloudZiti',
+            to: 'https://netfoundry.io/pricing/'
+          },
+          {
+            label: 'Blog',
+            to: 'https://blog.openziti.io'
           },
         ],
-        copyright: `Copyright © ${new Date().getFullYear()} NetFoundry Inc. Built with Docusaurus.`,
+        copyright: `Copyright © ${new Date().getFullYear()} NetFoundry, Inc.`,
```
Comment (dovholuknf) [inline docusaurus/docusaurus.config.js]: `NetFoundry Inc.` is correct. please fix

**Review by dovholuknf — APPROVED** [review]

**Pull request merged**